setup.sh

```
@@ -48,8 +48,22 @@ create_link() {
     fi
 }
 
-# --- 実行 ---
+# 1. 設定ファイルのリンク形成
 create_link "$OPCODE_SRC" "$OPCODE_DEST"
 create_link "$ALIASES_SRC" "$ALIASES_DEST"
 
+# 2. mise 環境の構築
+echo "\n--- mise environment setup ---"
```

Shellcheck (0.11.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)